### ISSUE TITLE
Confluence Skin no longer "compatible" with OSMC latest update (build 21.1.0) (Git 20240824-osmc)

### ISSUE THREAD

**Issue by LucidEye** [xbmc/skin.confluence#282]
My OSMC KODI running on a Raspberry Pi 4B just performed an automatic update, after the required reboot, and to my horror and sorrow, it removed my beloved Confluence skin because it said it "wasn't compatible".  It didn't even give me the option to roll-back :-(

I've been using the Confluence skin and only the Confluence skin since 2012 when KODI was still XBMC, it's the only interface I know and need and I really can't stand any of the other ugly and clunky interface skins, so is there any possibility of making Confluence once again compatible with the latest build of OSMC..... PPLLLEEEEAASSSE????

Version info taken from OSMC System Info screen...
OSMC build: 21.1 (21.1.0)
Git: 20240824-osmc
Compiled: 2024-08-24 

Thank you for keeping this classic interface alive for all of us nostalgia nerds!  :-)

**Comment by samnazarko** [comment]
Please start a forum post with logs on the OSMC forums so we can advise.

**Comment by PMac-UK** [comment]
I've landed here, looking for information about the confluence skin, as it wasn't supported after kodi update applied today, August 25th 2024.

**tl;dr**  -  reinstall confluence addon.


As per @LucidEye it's worked for a long time, and the platform asked for a reboot today for normal updates but confluence skin was disabled. 

I navigated through to original ui, updated the addon and after clicking 'use' chose to 'accept'.   But the app gave sad face blue screen and restarted with original ui.

After landing here, and seeing @bossanova808 feedback, it suggested to me some dirty data.
Therefore, I uninstalled 'confluence' addon.  It asked if wanted to remove some other settings info or such,  I said yes.
Then searched to install skin again,  found 'confluence'  it downloaded and offered to accept.
I did, and it's working fine.


All in all;  have kodi  remove the confluence skin,   then have it re-download and install it.   
Resolved issue for me.

**Comment by Hitcher** [comment]
Glad you got it sorted.

**Comment by Sopor** [comment]
I updated Confluence to the latest version and it was running fine until i restarted the Vero and it was back on OSMC skin.
What i did was to changed some settings when i was running Confluence.
I then rebooted and Confluence was my default skin again even after a reboot.
I don't know if this was a coincidence or not, but it worked 😊
